CHANGELOG.md

```diff
@@ -3,6 +3,41 @@
 List of the most important changes for each release.
 
 
+## 0.15.1
+
+### Overview of new features
+The goals of this release were to fix a bug preventing proper syncing of an individual user's data across multiple devices and to made some small frontend improvements
+
+#### Added
+
+  - Depreciation warnings for Python 3.4 and 3.5
+  - Added auto-alignment property for text display in cards, based on the language
+  - Allow untranslated headers in csv + correct serialization into json
+
+#### Changed
+
+  - Several morango bumps, now on latest version v0.6.8
+  - Bump zeroconf for fix to properly trigger service update events
+  - Bump KDS version to latest version v1.3.0
```

Comment:
```suggestion
  - Bump KDS version to v1.3.0
```

'latest version' will be false in the near future, so either remove or clarify to 'current latest version'